lib/index.js

```diff
@@ -5,8 +6,7 @@ var shouldRunTest = (testName, options) => {
   var exclude = options.exclude || [];
 
   if (options.device == 'mobile') {
-    exclude = new Set(exclude.concat(assertions.mobileExclusions));
-    exclude = [...exclude];
+    exclude = exclude.concat(assertions.mobileExclusions);
```

Comment:
the only functional difference here that might matter is that the `Set` removes dupes - perhaps importing a lodash unique module and using that?